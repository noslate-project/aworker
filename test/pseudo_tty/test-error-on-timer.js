'use strict';

setTimeout(() => {}, 2 ** 32);
