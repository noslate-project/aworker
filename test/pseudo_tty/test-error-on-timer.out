* does not fit into a 32-bit signed integer.
Timeout duration was set to 1. TimeoutOverflowWarning
